src/log/log.go

```diff
@@ -0,0 +1,34 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+const (
+	// ANSI color codes
+	Red    = "\033[31m"
+	Green  = "\033[32m"
+	Yellow = "\033[33m"
+	White  = "\033[97m"
+	Reset  = "\033[0m"
+)
+
+func Printf(color, format string, v ...interface{}) {
+	msg := fmt.Sprintf(format, v...)
+	log.Printf(color + msg + Reset)
+}
+
+func Println(color string, args ...interface{}) {
+	var msg string
+	for _, v := range args {
+		str, _ := v.(string)
+		msg += str
+	}
+	log.Println(color + msg + Reset)
+}
```

Comment:
The package can abstract the color setting complexity by unexporting Printf and Println and instead adding the following:
- `logger.Successf`, `logger.Warnf`, `logger.Errorf`
- `logger.Successln`, `logger.Warnln`, `logger.Errorln`

Which makes me wonder; isn't there already some Go library implementing these?

Reply:
> * `logger.Successf`, `logger.Warnf`, `logger.Errorf`
> * `logger.Successln`, `logger.Warnln`, `logger.Errorln`

Maybe also add `logger.Infof` and `logger.Infoln` for standard white output, or just use the standard `log.Println` and `log.Printf` in this case, which one is preferred? 

Regarding using another library, I found many libraries that already have this implemented, such as:
- [rs/zerolog](https://github.com/rs/zerolog)
- [uber-go/zap](https://github.com/uber-go/zap)
- [golog](https://git.kirsle.net/go/log)

And many others, but I found that many of them seem to be too bloated for what we want to use. Also, they usually print colorful output only on the tags (`[ERROR]`, `[INFO]`, `[WARN]` ...). Sure, maybe it can be configured—I didn't check all of them. But in any case, for this situation, the icons already used serve this function. The idea of printing the entire error, warning, or success line in a different color is both to draw the user's attention and to aid in reading.

Additionally, adding more dependencies leaves us with another library to keep track of and check if it is still maintained.

What do you think?

Reply:
Yes, the Info funcs are also needed.
A simple and well maintained library would be preferred, but let's stick with the 8 functions for now and maximize code reuse among them.